.github/workflows/daily_report_receipt.yml

```diff
@@ -15,6 +15,34 @@ on:
           - prod
         default: prod
 
+      time_range:
+        required: true
+        type: choice
+        description: Select the Time Range
+        options:
+          - daily
+          - weekly
+          - dozen
+          - monthly
+          - custom
+        default: weekly
```

Comment:
with default `weekly` we should change cron to `every monday at 06:00 a.m.`

Reply:
change applied